PokemonRampUp/Model/Pokemon.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Pokemon.swift
+//  PokemonRampUp
+//
+//  Created by Thushen Mohanarajah on 2023-02-10.
+//
+
+import Foundation
+import UIKit
+
+class Pokemon {
```

Comment:
you should use a struct for your models.

read more here: https://stackoverflow.com/questions/24232799/why-choose-struct-over-class

Reply:
This reading makes sense. From my understanding most languages strive to be pass-by-reference or pass-by-value. Why would Swift openly support both. In Java objects are mostly pass by pass-by-reference while primitive data types are pass-by-value